src/main/kotlin/com/jetbrains/snakecharm/stringLanguage/lang/SmkSLPythonRegExpInjector.kt

```diff
@@ -0,0 +1,30 @@
+package com.jetbrains.snakecharm.stringLanguage.lang
+
+import com.intellij.lang.injection.MultiHostRegistrar
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.elementType
+import com.jetbrains.python.codeInsight.PyInjectionUtil
+import com.jetbrains.python.codeInsight.PyInjectorBase
+import com.jetbrains.python.codeInsight.regexp.PythonRegexpLanguage
+import com.jetbrains.python.psi.PyStringLiteralExpression
+
+open class SmkSLPythonRegExpInjector : PyInjectorBase() {
```

Comment:
`SmkSL` means `Snakemake String formal language`,  is a fake language for python strings with snakemake handled injections, like: `input: "/foo/boo{suffix}.txt"`. This issue isn't related to SmkSL language, please rename this class and related classes if nee

Reply:
E.g. `SmkWildcardConstraintsRegExpInjector` sounds better to me

Reply:
Also please move it from `com.jetbrains.snakecharm.stringLanguage` to some other place, `stringLanguage` is reserved for SmkSL language in our project

Reply:
I changed it